Camera.hpp

```diff
@@ -69,6 +76,10 @@ namespace spic {
              */
             void AspectHeight(double newAspectHeight);
 
+            void Target(std::shared_ptr<RenderTarget> target);
+
+            std::shared_ptr<RenderTarget> Target() const;
+
```

Comment:
Is het echt nodig om een struct met x,y in een shared_ptr te zetten?

Reply:
Ja omdat we anders niet de render target op een nullptr kunnen zetten